core/controllers/reader.py

```diff
@@ -213,6 +213,25 @@ def get(self, exploration_id):
 
         exploration_rights = rights_manager.get_exploration_rights(
             exploration_id, strict=False)
+
+        # Retrieve all classifiers for the exploration.
+        state_classifier_mapping = {}
+        classifier_training_jobs = (
+            classifier_services.get_classifier_training_jobs(
+                exploration_id, exploration.version, exploration.states))
+        for index, state_name in enumerate(exploration.states):
+            if classifier_training_jobs[index] == None:
+                state_classifier_mapping[state_name] = None
+            else:
+                # Not making this a multi-get since we are deprecating
```

Comment:
No, this won't do. We can't do individual GETs in a for loop; it takes too long. This will block merge.

Why don't you deprecate ClassifierDataModel first and then do this PR?